content/guides/http/go/net_http/server.md

```diff
@@ -0,0 +1,231 @@
+
+```go
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"go.opencensus.io/plugin/ochttp"
+)
+
+func main() {
+	originalHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello, World!"))
+	})
+	och := &ochttp.Handler{
+		Handler: originalHandler, // The handler you'd have used originally
+	}
+
+	// Now use the instrumnted handler
+	if err := http.ListenAndServe(":9999", och); err != nil {
+		log.Fatalf("Failed to run the server: %v", err)
+	}
+}
+```
+
+### Traces
+The instrumented handler creates a trace automatically for each request that is received. We shall need a [Trace exporter](/exporters/supported-exporters/go), allowing us to examine the traces.
+If the request that it received contains a trace with the same context propagation format, it will continue the trace as a root, for example
+
+![](/images/ochttp-server-trace-non-link.png)
+
+However, if the handler is started with the field [IsPublicEndpoint](https://godoc.org/go.opencensus.io/plugin/ochttp#Handler.IsPublicEndpoint), then the span of the propagated span will be recorded as a [Parent Link](/tracing/span/link/) instead for example
+
+On the server side
+![](/images/ochttp-server-trace-link-server.png)
+
+which is the parent link of
+![](/images/ochttp-server-trace-link-client.png)
+
+
+### Metrics
+The instrumented handler records metrics automatically for each request that is received. We shall need a [Stats exporter](/exporters/supported-exporters/go), allowing us to examine the metrics.
+
+Metrics can be enabled by simply registering the [ochttp.DefaultServerViews](https://godoc.org/go.opencensus.io/plugin/ochttp#DefaultServerViews)
+```go
+        // In our main, register ochttp Server views
+        if err := view.Register(ochttp.DefaultServertViews...); err != nil {
+                log.Fatalf("Failed to register server views for HTTP metrics: %v", err)
+        }
+```
+
+which then provides the following metrics
+
+Metric|Prefix|Description|Tags|Aggregation
+---|---|---|---|---
+Requests count by method|"opencensus.io/http/server/request_count_by_method"|The number of requests received|"http.method"|Count
+Responses count by method|"opencensus.io/http/server/response_count_by_method"|The number of responses sent|"http.method"|Count
+Size distribution of HTTP request bodies|"opencensus.io/http/server/request_bytes"|The number of bytes received per request||Distribution
+Size distribution of HTTP response bodies|"opencensus.io/http/server/response_bytes"|The number of bytes sent per request||Distribution
+Server Latency|"opencensus.io/http/server/latency"|The latency distribution of HTTP responses||Distribution
```

Comment:
>this is already described in the spec. May be simply put a link to it.

Unfortunately that doesn't help with accessibility of information (sure it is duplication) but the goal of putting that here is so that they can find it easily in one place and follow the reading.
However, I'll keep the link in the references :)